### ISSUE TITLE
(Feature Request) Option To Only Show Enabled Breakpoints

### ISSUE THREAD

**Issue by KcHNST** [cyrus-and/gdb-dashboard#246]
Would it be possible to make a style option to only show the currently enabled breakpoints?

I'm thinking of something like:

`dashboard breakpoints -style disabled -- Hide disabled breakpoints.`

or 

`dashboard breakpoints -style onlyenabled -- Show only enabled breakpoints.`

**Comment by cyrus-and** [comment]
Yes, and I thought it was trivial at first but it's not, working on it...
